x-pack/test/functional/config.js

```diff
@@ -514,6 +514,14 @@ export default async function ({ readConfigFile }) {
           elasticsearch: {
             cluster: ['manage_pipeline', 'cluster:monitor/nodes/info'],
           },
+          kibana: [
```

Comment:
Did you see Larry's comment here: https://github.com/elastic/kibana/issues/102427#issuecomment-863172007? Does it work if you apply read access to advanced settings as suggested? It seems strange that `fleet` would be required here 🤔 .

```
          kibana: [
            {
              feature: {
                advancedSettings: ['read'],
              },
              spaces: ['*'],
            },
          ]
```

Reply:
Fixed.